packages/bruno-electron/src/ipc/collection.js

```diff
@@ -309,7 +309,7 @@ const registerRendererEventHandlers = (mainWindow, watcher, lastOpenedCollection
       if (!fs.existsSync(oldPath)) {
         throw new Error(`path: ${oldPath} does not exist`);
       }
-      if (fs.existsSync(newPath)) {
+      if (fs.existsSync(newPath) && path.basename(oldPath) === path.basename(newPath)) {
```

Comment:
While this works to allow to change the request's name when just changing a character case, it also allows for request to overwrite each other when renaming.

This problem has been discussed in #531 and I added a possible solution for this here: https://github.com/usebruno/bruno/pull/531#issuecomment-1888068214

This could be implemented like this:

```js
      if (fs.existsSync(newPath)) {
        // Try to resolve both paths, if they reference the same file we are on a case-insentive filesystem like NTFS and are trying to rename a file
        if (fs.statSync(newFilePath).ino !== fs.statSync(oldFilePath).ino) {
          throw new Error(`path: ${oldPath} already exists`);
        }
      }
```